FindOnPage.md

```diff
@@ -0,0 +1,653 @@
+        /// regardless of the previous active match. This behavior indicates that changing the find query initiates a 
+        /// completely new find session, rather than continuing from the previous match index. This distinction is essential 
+        /// to understand when utilizing the StartFind method for navigating through text matches.
+        Windows.Foundation.IAsyncAction StartFindAsync(CoreWebView2FindConfiguration configuration);
+
+
+        /// Navigates to the next match in the document.
+        /// If there are no matches to find, FindNext will wrap around to the first match if the search direction is forward, 
+        /// or to the last match if the search direction is backward.
+        /// If called when there is no find session active, FindPrevious will silently fail.
+        void FindNext();
+        
+        /// Navigates to the previous match in the document.
+        /// If there are no matches to find, FindPrevious will wrap around to the last match if the search direction is forward, 
+        /// or to the first match if the search direction is backward.
+        /// If called when there is no find session active, FindPrevious will silently fail.
+        void FindPrevious();
+
+
+        /// Stops the current 'Find' operation and hides the Find bar.
+        /// If called with no Find session active, it will silently do nothing.
+        void StopFind();
+
+        /// Retrieves the index of the currently active match in the find session. Returns the index of the currently active match, or -1 if there is no active match.
+        Int32 ActiveMatchIndex { get; };
+
+        /// Gets the total count of matches found in the current document based on the last find sessions criteria. Returns the total count of matches.
+        Int32 MatchCount { get; };
+        
+        /// Registers an event handler for the MatchCountChanged event. 
+        /// This event is raised when the total count of matches in the document changes due to a new find operation or changes in the document. 
+        /// The parameter is the event handler to be added. Returns a token representing the added event handler. This token can be used to unregister the event handler.
+        [event_handler("", "", "")]
+        event Windows.Foundation.TypedEventHandler<CoreWebView2Find, Object> MatchCountChanged;
+
+        /// Registers an event handler for the ActiveMatchIndexChanged event. This event is raised when the index of the currently active match changes.
+        /// This can happen when the user navigates to a different match or when the active match is changed programmatically. 
+        /// The parameter is the event handler to be added. Returns a token representing the added event handler. 
+        /// This token can be used to unregister the event handler.
+        [event_handler("", "", "")]
+        event Windows.Foundation.TypedEventHandler<CoreWebView2Find, Object> ActiveMatchIndexChanged;
+    };
```

Comment:
Missing the CoreWebView2.Find property in this MIDL3 and the CoreWebView2Environment.CreateFindConfiguration in this MIDL3